writer.go

```diff
@@ -23,15 +23,14 @@ import (
 	"strings"
 )
 
-// A Writer writes an ach.file to a NACHA encoded file.
-//
-// As returned by NewWriter, a Writer writes ach.file structs into
-// NACHA formatted files.
-//
+// Writer writes a File to an io.Writer.
```

Comment:
Updated the GoDoc here for clarity. The previous code comments were a bit redundant.